templates/repo/scripts/failed_output_only

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
I was trying to get this to work to suppress non-error output when running in atlantis. I didn't quite get it working, but going to leave it here to iterate on later.

Reply:
maybe leave a TODO on the file for future selves